index.html

```diff
@@ -302,12 +302,12 @@ <h4 class="let-connect">LET'S CONNECT</h4>
         </div>
       </section>
       <section class="contact-me-form">
-        <h2>Contact me</h2>
-        <p>if you have an application you are intersted in developing, a feature that you need build or a project that needs coding.I'd love to help with it</p>
-        <input type="text" id="name" name="user_name">
-        <input type="email" id="email" name="user_email">
-        <textarea name="user_msg" id="msg"></textarea>
-        <button type="submit" id="button">Get it Touch</button>
+        <h2 class="contact-t">Contact me</h2>
+        <p class="contact-m">if you have an application you are intersted in developing, a feature that you need build or a project that needs coding.I'd love to help with it</p>
+        <input class="input" type="text" id="name" name="user_name">
+        <input class="input" type="email" id="email" name="user_email">
+        <textarea class="input textarea" name="user_msg" id="msg">Write your message here.</textarea>
+        <button class="button-submit get-my-resume-btn" type="submit" id="button">Get it Touch</button>
```

Comment:
Good structure,only one comment the please change p  tage data to have the same struture as the figma design may be add a break b/n the lines